### DISCUSSION TITLE
Orientation tracking available?

### DISCUSSION THREAD

**Discussion by wmsheer** [DeniseCaiLab/ezTrack#65] [General]
Hi,

I just re-watched the video from last fall's Miniscope Workshop covering ezTrack, and at one point Zach mentioned a beta test version that included orientation tracking capabilities. The most recent release on GitHub doesn't seem to support orientation tracking (unless I'm missing something) - is there a way to get access to this version?

Thanks!
Will

**Comment by ZachPenn** [comment]
Hi Will,

A beta version is actually available. Unfortunately, I just haven't had anyone reach out to me about doing location tracking so it's sort of been on stand-still.  If you're interested in trying it out I'd be happy to advise.  It definitely works, but I've been wanting more people to test it out and give feedback on what they want before putting on the master branch.  Currently, if you go to the main eztrack github page and select the tab at the top left that says `master` (the branch), there will be a drop-down menu where you can select `orientation_dev`.  There is some documentation so if you run through `LocationTracking_Individual.ipynb` you should be able to try it out pretty easily.

**Comment by wmsheer** [comment]
Great! I'll download that version and give it a shot. Will loop back here with any issues I run into, etc. Thanks so much!

**Reply by ZachPenn** [reply]
Sounds good.  Note that the conda environment is a bit different (other packages required).  Install instructions are on the main page of the `orientation_dev` branch.
